tests/utils/test_session.py

```diff
@@ -53,3 +55,13 @@ def test_provide_session_with_kwargs(self):
 
         session = object()
         assert wrapper(session=session) is session
+
+    @pytest.mark.asyncio
+    async def test_async_session(self):
+        from airflow.settings import create_async_session
+
+        session = create_async_session()
+        session.add(Log(event="hihi1234"))
+        await session.commit()
+        l = await session.scalar(select(Log).where(Log.event == "hihi1234").limit(1))  # noqa: E741
+        assert l.event == "hihi1234"
```

Comment:
```suggestion
        log_result = await session.scalar(select(Log).where(Log.event == "hihi1234").limit(1))  # noqa: E741
        assert log_result.event == "hihi1234"
```

Reply:
i can do this in a followup kaxil if it pleases you. don't want to wait for 99 ci checks to run agaih 😬

Reply:
haha I definitely understand -- and its a nit so will leave it up to you :)